### DISCUSSION TITLE
[Feature] Support ICO format

### DISCUSSION THREAD

**Discussion by basicfu** [immich-app/immich#13025] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The format I exported from Google Photos includes a favicon.ico icon, but I am unable to import immich. Uploading the file has no response, and I did not see the format image/x-icon in the source code
https://github.com/immich-app/immich/blob/main/server/src/utils/mime-types.ts

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I'll leave the request open, but note that a `favicon.ico` is probably not a file you actually care about. You can try opening it in your browser to view it.

**Reply by basicfu** [reply]
I may collect some. ico files, Perhaps it's not just favicon.ico,
ico is also a type of image format, and I believe that immich, as a professional software for managing images, should support this image format
